SIS_import/sis_set_url.rb

```diff
@@ -271,8 +272,17 @@ def getMPathwayTerms(esbToken)
 	call_url = @esbUrl + "/Curriculum/SOC/v1/Terms";
 	result= ESB_APICall(call_url, "Bearer " + esbToken, "application/json", "GET", nil)
 	if (!result.nil?)
-		# an array returned here event
-		result["getSOCTermsResponse"]["Term"].each do |term|
+		# ideally the Term element should always be an Array
```

Comment:
I think it would be clearer to pull out the term element into a variable and then process the contents of the variable.  The only change then would be to wrap the value in the variable in a list if that isn't already a list.  This would lead to less repeated code.